### DISCUSSION TITLE
"Can't set headers after they are sent to the client" in relation to express-session

### DISCUSSION THREAD

**Discussion by mpgr04** [expressjs/session#1044] [Q&A]
Hi,
I use express-session in my application.
During the development process I set the secure option of the session cookie to false, because I don't use https locally.

But as soon as I set this option to true and want to use the application on my webserver which has https, the following error occurs:
Can't set headers after they are sent to the client

However, this only occurs on the web server and not locally.

I have already checked my code to see if multiple responses are sent in one request, but this is not the case.

I am grateful for any help!

**Comment by dougwilson** [comment]
Hello, I'm sorry your issue sat without response. Most likely no one knew the answer off-hand. I can help, though I'm not sure either. Can you provide some way that we can reproduce the error and debug it?
